proposals/dao/swap_dpi_to_dai.ts

```diff
@@ -0,0 +1,172 @@
+import { ethers } from 'hardhat';
+import { expect } from 'chai';
+import { TransactionResponse } from '@ethersproject/providers';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner, overwriteChainlinkAggregator, time } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+const toBN = ethers.BigNumber.from;
+
+/*
+Swap DPI for DAI using an LBP Swapper
+*/
+
+const fipNumber = 'swap_dpi_to_dai';
+
+// LBP Swapper config
+const LBP_FREQUENCY = 86400 * 30; // 1 month in seconds
+const MIN_LBP_SIZE = ethers.constants.WeiPerEther.mul(10_000); // 10k
+let poolId; // auction pool id
+
+const daiSeedAmount = toBN('187947000000000000000000');
+const dpiSeedAmount = toBN('37888449801955370645659');
+
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  // Amounts:
+  // DPI: 37888449801955370645659 (95%), 37k DPI, $3,758,957.86
+  // DAI: 187947000000000000000000 (5%), 187k DAI, $187,947.89
+
+  // 1. Deploy the Balancer LBP swapper
+  const BalancerLBPSwapperFactory = await ethers.getContractFactory('BalancerLBPSwapper');
+  const dpiToDaiSwapper = await BalancerLBPSwapperFactory.deploy(
+    addresses.core,
+    {
+      _oracle: addresses.chainlinkDpiUsdOracleWrapper,
```

Comment:
did you check the oracle's direction. in other words what is the initial swap price

Reply:
I think its fine, and can be changed if need be later

Reply:
Yep I did, it reports the price of DPI in terms of DAI. It currently reports `96216077210000000000` which is `96.2 DAI` - close to the coingecko DPI price of ~$96.12